monitoring/influxdb.go

```diff
@@ -43,6 +44,8 @@ const (
 	batchWriteIntervalParamName = "batch-write-interval"
 	bufferSizeParamName         = "buffer-size"
 	maxBufferSizeParamName      = "max-buffer-size"
+
+	serviceNameParamName = "service-name"
```

Comment:
It seems the default value `service-name` isn't quite useful. How about do set it at all by default?

Reply:
it's param name, not a default value.